app-modules/interaction/src/Filament/Resources/InteractionInstitutionResource.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Assist\Interaction\Filament\Resources;
+
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Forms\Components\TextInput;
+use Assist\Interaction\Models\InteractionInstitution;
+use Assist\Interaction\Filament\Resources\InteractionInstitutionResource\Pages\EditInteractionInstitution;
+use Assist\Interaction\Filament\Resources\InteractionInstitutionResource\Pages\ListInteractionInstitutions;
+use Assist\Interaction\Filament\Resources\InteractionInstitutionResource\Pages\CreateInteractionInstitution;
+
+class InteractionInstitutionResource extends Resource
```

Comment:
I may be wrong, but I think that this should be tied to the already existing concept of `Institution,` right? We don't have to change it now, especially with the concept being in flux based on what Product has discussed with us.

Reply:
In our last discussion on this, we decided that for now this just needs to be something configurable by admins, and the connection between these institutions and what we already have an an established model for needs to be explored further.